http3/server.go

```diff
@@ -446,7 +441,7 @@ func (s *Server) handleConn(conn quic.Connection) error {
 			}
 			return fmt.Errorf("accepting stream failed: %w", err)
 		}
-		go func() {
+		go func(str quic.Stream) {
```

Comment:
because it's a in a for loop we need to pass the conn as a value to the go-routine to avoid race conditions

Reply:
Same here.